src/components/pages/OnboardingServiceProvider/OnboardingServiceProvider.tsx

```diff
@@ -0,0 +1,175 @@
+  Tab,
+  TabPanel,
+} from '@catena-x/portal-shared-components'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import LinkIcon from '@mui/icons-material/Link'
+import { Box } from '@mui/material'
+import './OnboardingServiceProvider.scss'
+
+const OnboardingServiceProvider = () => {
+  const { t } = useTranslation()
+  const [activeTab, setActiveTab] = useState<number>(0)
+
+  const handleTabChange = (
+    _e: SyntheticEvent<Element, Event>,
+    value: number
+  ) => {
+    setActiveTab(value)
+  }
+
+  const getTabsIcon = (step: number) => {
+    return (
+      <Typography
+        variant="label3"
+        sx={{
+          background: activeTab + 1 === step ? '#0f71cb' : 'white',
+          color: activeTab + 1 === step ? 'white' : '#0f71cb',
+          outline: '2px solid #0f71cb',
+          flex: '0',
+          marginRight: '20px',
+          borderRadius: '50%',
+          height: '20px',
+          width: '20px',
+          minWidth: '20px',
+          textAlign: 'center',
+          lineHeight: '20px',
+          position: 'relative',
+        }}
+      >
+        {step}
+      </Typography>
+    )
+  }
+  return (
+    <main className="onboarding-service-page-container ">
+      <section>
+        <div className="onboarding-service-header">
+          <Typography variant="h2" className="onboarding-service-title">
+            {t('content.onboardingServiceProvider.headertitle')}
+          </Typography>
+          <Typography variant="body2" className="onboarding-service-desc">
+            {t('content.onboardingServiceProvider.desc')}
+          </Typography>
+          <Box
+            sx={{
+              position: 'relative',
+              height: '100%',
+              width: '90%',
+              background: '#F4FBFD',
+              margin: 'auto',
+            }}
+          >
+            <IconButton
+              aria-label="close"
+              onClick={() => {}}
+              sx={{
+                left: '0',
+                position: 'absolute',
+                top: '50%',
+                msTransform: 'translateY(-50%)',
+                transform: 'translateY(-50%)',
+              }}
+            >
+              <LinkIcon />
+            </IconButton>
+            <Box sx={{ textAlign: 'center' }}>
+              <Typography variant="body3">
+                {t('content.onboardingServiceProvider.subDesc1')}
+              </Typography>
+              <Typography variant="body3">
+                {t('content.onboardingServiceProvider.subDesc2')}
+              </Typography>
+            </Box>
+            <IconButton
+              aria-label="close"
+              onClick={() => {}}
+              sx={{
+                right: '0',
+                position: 'absolute',
+                top: '50%',
+                msTransform: 'translateY(-50%)',
+                transform: 'translateY(-50%)',
+              }}
+            >
+              <ArrowForwardIcon />
+            </IconButton>
```

Comment:
this block is very similar to the one in L87-L99 - I suggest you create a local component on top of the file and just specify the child and right / left